modules/hydropower/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2018-2021 Tim Tröndle (ETH Zürich / IASS Potsdam), Bryn Pickering (ETH Zürich)
+Copyright (c) 2024 AUTHORS
```

Comment:
If the module was taken from EC, attribution is given to the original authors, and then to the authors at `ec_modules`.

Reply:
`-2022` and does that assume 2022-2024 is copyright-free?

Reply:
I'll ask the license expert about this tomorrow. I assume it does not really matter.
Still, I've added those years.